src/main/java/com/spotify/docker/AbstractDockerMojo.java

```diff
@@ -246,25 +252,35 @@ protected AuthConfig authConfig() throws MojoExecutionException, SecDispatcherEx
         }
 
         return authConfigBuilder.build();
-      } else if (useConfigFile != null && useConfigFile){
-
-          final AuthConfig.Builder authConfigBuilder;
-          try {
-            if (!isNullOrEmpty(registryUrl)) {
-              authConfigBuilder = AuthConfig.fromDockerConfig(registryUrl);
-            } else {
-              authConfigBuilder = AuthConfig.fromDockerConfig();
-            }
-          } catch (IOException ex){
-            throw new MojoExecutionException(
-                      "Docker config file could not be read",
-                      ex
-            );
+      } else if (useConfigFile != null && useConfigFile) {
```

Comment:
If we want to always fall back to using `.docker`, perhaps just remove this else branch and the `useConfigFile` switch?